templates/index/hero.html

```diff
@@ -1,4 +1,4 @@
-{{ define "hero" }}
+{{ define "hero_old" }}
```

Comment:
If I am not mistaken, this is not used anywhere and should be deleted. The same applies to `hero_css_old`.